src/index.js

```diff
@@ -16,7 +16,8 @@ const codes = /** @type {import('./types').CodeNameMap} */({})
 
 // eslint-disable-next-line guard-for-in
 for (const key in names) {
-  codes[names[key]] = key
+  const name = /** @type {HashName} */(key)
```

Comment:
the parens are interesting, is this being mandated by the linter or the type checker?

Reply:
its how you cast to another type https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#casts

Reply:
👍 interesting